gr-utils/modtool/templates/gr-newmod/docs/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@
 # Setup dependencies
 ########################################################################
 find_package(Doxygen)
+find_package(MathJax)
```

Comment:
This line needs to be removed. It will try to find FIndMathJax2.cmake in the OOT, which is not present.

Unfortunately, I don't have an easy fix for this yet.

Reply:
Are there even OOTs out there that make use of LaTeX in their Doxygen manual?  Important thing is to fix it within GR, having it work within OOTs would be nice but it shouldn't block the GR side imo.

Reply:
That would be one way to solve the issue.